### PR TITLE
feat: Create `Price` component

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to create `Price` component on `@faststore/components` and update all references in `@faststore/ui`.

- [x] Move `Price` component from `@faststore/ui` to `@faststore/components`
- [x] Update `Price` docs.

## How to test it?

Just run `yarn workspace docs start` and check the `Price` component.

### CODE REVIEW

**Review by eduardoformiga — COMMENTED** [review]
Nice! 
Testing using the [preview link](https://faststore-6z41ei8h3-faststore.vercel.app/reference/ui/atoms/Price), I think we need to adjust some things
Screen Shot 2022-12-07 at 17 12 44

**Review by eduardoformiga — APPROVED** [review]
Looks good!
[preview](https://faststore-ngpwq8wij-faststore.vercel.app/reference/ui/atoms/Price)

File: packages/components/src/atoms/Price/Price.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
         ref={ref}
         data-fs-price
         data-testid={testId}
-        data-variant={variant}
+        data-fs-variant={variant}
```
Comment (hellofanny) [inline packages/components/src/atoms/Price/Price.tsx:56]:
```suggestion
        data-fs-price-variant={variant}
```


WTYT to make it more specific?

File: apps/docs/docs/reference/ui/molecules/Tag.mdx [hunk 2]
```diff
@@ -52,7 +52,7 @@ import '@faststore/ui/src/components/molecules/Tag/styles.scss'
 
 <TokenTable>
   <TokenRow token="--fs-tag-text-color" value="var(--fs-color-text)" />
-  <TokenDiver />
+  <TokenDivider />
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/Tag.mdx:55]: niceee catch! 💯

File: apps/docs/docs/reference/ui/atoms/Price.mdx [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
+  <Price value={62.5} variant="savings" />
+  <Price value={62.5} variant="installment" />
+</div>
+```
+
+---
+
 ## Import
 
 ```tsx
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/atoms/Price.mdx:30]:
```suggestion
```tsx
import { Price } from '@faststore/ui'
```

Import Styles

```tsx
import '@faststore/ui/src/components/atoms/Price/styles.scss'
```
```

Comment (hellofanny) [inline apps/docs/docs/reference/ui/atoms/Price.mdx:30]: Can you also add styles import here pls.


```
```tsx
import '@faststore/ui/src/components/atoms/Price/styles.scss'
```
```

File: apps/docs/docs/reference/ui/atoms/Price.mdx [hunk 4]
```diff
@@ -1,15 +1,30 @@
...
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
 # Price
 
 Prices are used to display the price of a given product, discount and total values.
 
+## Overview
```
Comment (renatamottam) [inline apps/docs/docs/reference/ui/atoms/Price.mdx:14]: I guess the sections order are a bit off. I would bring props and design tokens up.

**Review by renatamottam — APPROVED** [review]
Nice work, feijs!

**Pull request merged**